v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -89,6 +89,8 @@ spec:
          - exposedPort: 13132
          - exposedPort: 13133
      memoryLimit: "512M"
+     cpuLimit: "1000m"
+     cpuRequest: "30m"
```

Comment:
Not sure we should run theia with 1/3rd of a single CPU.

Reply:
hmm.. not sure I understand is not 1/3rd it is about requesting only 3% of the core with a 1 full core limit

Reply:
It's even less than that. It's 0.03 of a CPU.